addons/backpacks/anprc155/CfgVehicles.hpp

```diff
@@ -17,8 +17,8 @@ class TFAR_anprc155: TFAR_Bag_Base {
 HIDDEN_CLASS(tf_anprc155 : TFAR_anprc155); //#Deprecated dummy class for backwards compat
 class TFAR_anprc155_coyote: TFAR_anprc155 {
     author = "Raspu, Gandi, Nkey";
-    displayName = "AN/PRC-155 Coyote";
```

Comment:
@dedmen I remaine some **displayName** still in stringtable, because I think these name can be translate.(Like Big or Small, and colors or camo etc...)